src/librustdoc/html/render.rs

```diff
@@ -2211,9 +2207,11 @@ impl<'a> fmt::Display for Sidebar<'a> {
     fn fmt(&self, fmt: &mut fmt::Formatter) -> fmt::Result {
         let cx = self.cx;
         let it = self.item;
+        let parentlen = cx.current.len() - if it.is_mod() {1} else {0};
+
+        // this is not rendered via JS, as that would hamper the accessibility
```

Comment:
The mention of accessibility is quite off, I admit. (My original intention is that, we should preserve as much HTML as possible unless that will result in bloat.) I'll reword that.